api/DocumentFragment.json

```diff
@@ -95,200 +95,6 @@
             "deprecated": false
           }
         }
-      },
-      "methods": {
```

Comment:
What in the world in this? Is there more of this kind or grouping in BCD?

Reply:
I hope not! 😢